index.html

```diff
@@ -0,0 +1,131 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
```

Comment:
I think a viewport meta tag should be included. This tag gives the browser instructions on how to control the page's dimensions and scaling